src/assets/css/_sass/elements/_page.scss

```diff
@@ -33,4 +31,40 @@ body {
 
 main {
   flex: 1 0 auto; // 3
-}
+  .hero {
+    height: 100vh;
+    background: url('/assets/img/trees.png');
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    color: $color-text--light;
+    .title__container {
+      display: flex;
+      flex-direction: column;
+      padding-left: 60px; 
+      width: 500px;
+      line-height: normal;
```

Comment:
remove `line-height: normal;` solve what the problem is instead of overriding it